tools/tuf/releaser.sh

```diff
@@ -132,19 +132,18 @@ release_fleetd_to_edge () {
     BRANCH_NAME="release-fleetd-v$VERSION"
     ORBIT_TAG="orbit-v$VERSION"
     if [[ "$SKIP_PR_AND_TAG_PUSH" != "1" ]]; then
-        prompt "A PR will be created to trigger a Github Action to build desktop."
+        prompt "A PR will be created for the release of fleetd $VERSION, and a tag will be pushed to trigger a Github Action to build desktop and orbit."
         pushd "$GIT_REPOSITORY_DIRECTORY"
         git checkout -b "$BRANCH_NAME"
         make changelog-orbit version="$VERSION"
         ORBIT_CHANGELOG=orbit/CHANGELOG.md
         "$GO_TOOLS_DIRECTORY/replace" .github/workflows/generate-desktop-targets.yml "FLEET_DESKTOP_VERSION: .+\n" "FLEET_DESKTOP_VERSION: $VERSION\n"
         git add .github/workflows/generate-desktop-targets.yml "$ORBIT_CHANGELOG"
         git commit -m "Release fleetd $VERSION"
-        git push origin "$BRANCH_NAME"
-        gh pr create -f -B main -t "Release fleetd $VERSION"
-        prompt "A 'git tag' will be created to trigger a Github Action to build orbit."
         git tag "$ORBIT_TAG"
-        git push origin "$ORBIT_TAG"
+        git push origin "$BRANCH_NAME" --follow-tags
+        # Create a new PR with the changelog.
+        gh pr create -f -B main -t "Release fleetd $VERSION"
```

Comment:
Do we need to create a branch `$BRANCH_NAME` to tag orbit?

Reply:
Nope!  Now it will just tag whatever branch is checked out.